### PR TITLE
chore: dd-trace upgrade to v1

### PR DESCRIPTION
**Pull request by ovasdi**
The type of this PR is: **Chore**

This PR upgrades [dd-trace](https://github.com/DataDog/dd-trace-js/tree/v1.7.1) library to `v1.7.1`

### Description

This is the first PR to upgrade dd-trace library to a more recent version. v1.7.1 is at _end of life_. Making these upgrades in smaller increments. Bump to v2 will come next.

Tested locally by running the dd-agent in docker and [generated some metrics](https://app.datadoghq.com/apm/services/force/operations/express.request/resources?env=none&resources=qson%3A%28data%3A%28visible%3A%21t%2Chits%3A%28selected%3Atotal%29%2Cerrors%3A%28selected%3Atotal%29%2Clatency%3A%28selected%3Aavg%29%2CtopN%3Aall%29%2Cversion%3A%210%29&summary=qson%3A%28data%3A%28visible%3A%21t%2Cerrors%3A%28selected%3Acount%29%2Chits%3A%28selected%3Aversion_count%29%2Clatency%3A%28selected%3Alatency%2Cslot%3A%28agg%3A75%29%2Cdistribution%3A%28isLogScale%3A%21f%29%29%2Csublayer%3A%28slot%3A%28layers%3Aservice%29%2Cselected%3Aabsolute%29%29%2Cversion%3A%211%29&topGraphs=latency%3Alatency%2CbreakdownAs%3Apercentage%2Cerrors%3Aversion_count%2Chits%3Aversion_count&start=1682623399527&end=1682623658150&paused=true).

[GRO-434]: https://artsyproduct.atlassian.net/browse/GRO-434?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: hokusai/development.yml [hunk 1]
```diff
@@ -14,7 +14,7 @@ services:
     env_file:
       - ../.env.shared
       - ../.env
-    command: ["yarn", "start:dev"]
+    command: ["yarn", "start"]
```
Comment (ovasdi) [inline hokusai/development.yml:17]: While testing this ended up running Force via `hokusai dev start` and turns out the command is out of date.

**Comment by ovasdi** [comment]
cc @joeyAghion

**Review by mc-jones — APPROVED** [review]
Let's give it a whirl!

File: package.json [hunk 2]
```diff
@@ -113,7 +113,7 @@
     "core-js": "2.6.5",
     "csurf": "^1.9.0",
     "currency.js": "1.2.1",
-    "dd-trace": "0.36.2",
+    "dd-trace": "1.7.1",
```
Comment (joeyAghion) [inline package.json:116]: Zooooom

**Pull request merged**